packages/router-plugin/src/router.module.ts

```diff
@@ -3,13 +3,13 @@ import { NgxsModule } from '@ngxs/store';
 import { DefaultRouterStateSerializer, RouterStateSerializer } from './serializer';
 import { RouterState } from './router.state';
 
+// NOTE: Must mark as `dynamic` due to
+// https://github.com/dherges/ng-packagr/issues/767
+export const NgxsModuleRouterState = NgxsModule.forFeature([RouterState]);
```

Comment:
This refactor broke backwards compatibility to Angular 5.
It has been reverted in PR #852 
See commit: https://github.com/ngxs/store/pull/852/commits/02d0a2866be0eb98eeff72fe2433edffcb809c9d